NetworkPkg/Ip6Dxe/Ip6Nd.c

```diff
@@ -2517,8 +2525,16 @@ Ip6ProcessRedirect (
   //
   OptionLen = (UINT16)(Head->PayloadLength - IP6_REDITECT_LENGTH);
   if (OptionLen != 0) {
+    // MU_CHANGE [BEGIN] - CodeQL change

```

Comment:
what am i missing here?  this looks like a useless add

Reply:
Discussed offline and pushed a fix, resolving this comment